packages/ember/src/components/form/radio-group.gts

```diff
@@ -0,0 +1,112 @@
+import { hash } from '@ember/helper';
+import { on } from '@ember/modifier';
+import { action } from '@ember/object';
+import Component from '@glimmer/component';
+
+import BoundValue from './bound-value.ts';
+
+import type { Optional } from '../../types';
+import type { ComponentLike } from '@glint/template';
+
+export interface RadioGroupFieldSignature {
+  Element: HTMLDivElement;
+  Args: {
+    name: string;
+    basic?: boolean;
+    disabled?: boolean;
+  };
+  Blocks: {
+    default: [{ Radio: ComponentLike<RadioField> }];
+  };
+}
+
+export default class RadioGroupField extends BoundValue<
+  RadioGroupFieldSignature,
+  string
+> {
+  @action
+  change(updatedValue: Optional<string>) {
+    if (updatedValue) {
+      this.value = updatedValue;
+      this.onChange?.(updatedValue);
+    }
+  }
+
+  get classList() {
+    const classes = ['form-control'];
```

Comment:
Is this used or necessary? The `basic` option doesn't seem to apply to radio buttons.